DataBaseTutorial/Migrations/DatabaseContextModelSnapshot.cs

```diff
@@ -0,0 +1,86 @@
+        {
+#pragma warning disable 612, 618
+            modelBuilder
+                .HasAnnotation("ProductVersion", "3.1.4");
+
+            modelBuilder.Entity("DataBaseTutorial.Product", b =>
+                {
+                    b.Property<int>("ProductId")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("INTEGER");
+
+                    b.Property<bool>("ProductAddled")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<DateTime>("ProductBuyedDate")
+                        .HasColumnType("TEXT");
+
+                    b.Property<string>("ProductName")
+                        .HasColumnType("TEXT");
+
+                    b.Property<double>("ProductWeight")
+                        .HasColumnType("REAL");
+
+                    b.HasKey("ProductId");
+
+                    b.ToTable("Product");
+
+                    b.HasData(
+                        new
+                        {
+                            ProductId = 1,
+                            ProductAddled = true,
+                            ProductBuyedDate = new DateTime(2020, 2, 12, 0, 0, 0, 0, DateTimeKind.Unspecified),
+                            ProductName = "Onion",
+                            ProductWeight = 0.5
+                        },
+                        new
+                        {
+                            ProductId = 2,
+                            ProductAddled = true,
+                            ProductBuyedDate = new DateTime(2020, 2, 12, 0, 0, 0, 0, DateTimeKind.Unspecified),
+                            ProductName = "Salat",
+                            ProductWeight = 1.0
+                        },
+                        new
+                        {
+                            ProductId = 3,
+                            ProductAddled = true,
+                            ProductBuyedDate = new DateTime(2020, 2, 12, 0, 0, 0, 0, DateTimeKind.Unspecified),
+                            ProductName = "Pickle",
+                            ProductWeight = 2.5
+                        },
+                        new
+                        {
+                            ProductId = 4,
+                            ProductAddled = true,
+                            ProductBuyedDate = new DateTime(2020, 2, 12, 0, 0, 0, 0, DateTimeKind.Unspecified),
+                            ProductName = "Potato",
+                            ProductWeight = 8.5600000000000005
+                        },
+                        new
+                        {
+                            ProductId = 5,
+                            ProductAddled = false,
+                            ProductBuyedDate = new DateTime(2020, 2, 12, 0, 0, 0, 0, DateTimeKind.Unspecified),
+                            ProductName = "Tomato",
+                            ProductWeight = 0.050000000000000003
+                        });
```

Comment:
Create a separate method that returns a list of seed data and use it's result in b.HasData(methodResult); It increases readability